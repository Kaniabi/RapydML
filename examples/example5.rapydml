def subsection($title):
  div(class="sub-section"):
    $title
  div(class="section-content")

div(id="main"):
  subsection('Foo'):
    'This is the foo section, it only has text'
  subsection('Bar'):
    img(src="image.png")
    div:
      'Bar section has images, and sub-children'
