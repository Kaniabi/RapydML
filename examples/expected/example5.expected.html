<div id="main">
  <div class="sub-section">
    Foo
  </div>
  <div class="section-content">
    This is the foo section, it only has text
  </div>
  <div class="sub-section">
    Bar
  </div>
  <div class="section-content">
    <img src="image.png" />
    <div>
      Bar section has images, and sub-children
    </div>
  </div>
</div>
